rclpy/test/test_logging.py

```diff
@@ -359,6 +359,22 @@ def test_clear_config(self):
             rclpy.logging._root_logger.get_effective_level(),
             my_logger.get_effective_level())
 
+    def test_logging_severity_from_string(self):
+        log_levels = {
+            'UNSET': rclpy.logging.LoggingSeverity.UNSET,
+            'DEBUG': rclpy.logging.LoggingSeverity.DEBUG,
+            'INFO': rclpy.logging.LoggingSeverity.INFO,
+            'WARN': rclpy.logging.LoggingSeverity.WARN,
+            'ERROR': rclpy.logging.LoggingSeverity.ERROR,
+            'FATAL': rclpy.logging.LoggingSeverity.FATAL
+        }
+        for level, severity in log_levels.items():
+            self.assertEqual(rclpy.logging.get_logging_severity_from_string(level), severity)
```

Comment:
@BMarchi `rclpy.logging.LoggingSeverity` is an [`IntEnum`](https://docs.python.org/3.6/library/enum.html#enum.IntEnum), you can do:

```python
for severity in rclpy.logging.LoggingSeverity:
    self.assertEqual(rclpy.logging.get_logging_severity_from_string(severity.name), severity)
```